content/java/developer-guide/symbology/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Symbology"
+date: 2017-07-13T23:24:01-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: "java"
+projectname: "WorldWind Java"
+layout: "project"
+---
+
+## Symbology
+
+The WorldWind API enables applications to display common symbology sets. Currently supported is [MIL-STD-2525](http://worldwind.arc.nasa.gov/milstd2525c/Mil-STD-2525C.pdf).
```

Comment:
Cleanest options:

1. Override the list template and use a standard page, formatting as needed. This is how the [Java Develop Guide page](https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/feature/section-based-templates/content/java/developer-guide/_index.md) is handled. Positive: explicit control of the ordering and style of the page. Negative: when additional content is added, it will not automatically be added to the list or index page.

2. Utilize Hugo's [default content ordering](https://gohugo.io/templates/lists/#order-content), which will first look for a `weight` front matter variable, then use the date, and title. This method is demonstrated with the symbology in the [feature/list-ordering-examples](https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/feature/list-ordering-examples/layouts/java/list.html#L39). The change from the current list ordering looks like this:

from:
```
{{ range .Data.Pages.ByTitle }}
```

to:
```
{{ range .Data.Pages }}
```

Positives: explicit ordering accomplished using built in variable (weight), and index page automatically updates when content is added. Negative: requiring the content to use the same date (simplified with the use of [archetypes](https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/feature/list-ordering-examples/archetypes/android.md#L4)).

Once an approach is finalized, either option can propagate and be used throughout the website.